yelyzaveta.kozhevnikova/task1.rb

```diff
@@ -0,0 +1,11 @@
+arr = Array.new(3) { Array.new(3) { rand(-10..10) } }
+pp(arr)
+res = 0
+(0..2).each do |x| 
```

Comment:
it's a bit better, but still, you didn't apply the original intent of my request. U are iterating using static numbers instead of dealing with the actual array.
```
arr.flatten.each do |value|
  if value.positive? 
    puts value 
    res += value  
  end
end
```

even tho it's possible to avoid `res` variable here by using `select` method:

```
arr = Array.new(3) { Array.new(3) { rand(-10..10) } }
pp arr.flatten.select { |value| value.positive? }.each { |value| pp value }.sum
```

Reply:
Okay thanks